spec/requests/users_spec.rb

```diff
@@ -1,7 +1,41 @@
 require 'rails_helper'
 
-RSpec.describe "Users", type: :request do
-  describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+RSpec.describe 'UsersController', type: :request do
+  let!(:user) { User.create(name: 'John Doe') }
+
+  describe 'GET #index' do
+    before do
+      get users_path
+    end
+
+    context 'Applies all test to index template' do
+      it 'should check if the response status is correct' do
+        expect(response).to have_http_status(200)
+      end
+      it 'should check if a correct template was rendered' do
+        expect(response).to render_template(:index)
+      end
+      it 'should check if the response body includes the correct placeholder text' do
+        expect(response.body).to include('Here is a list of users')
```

Comment:
- [ ] This test is also failing. This should be "list of users". Or you can modify the view page to match the test you are expecting.